src/v/pandaproxy/rest/configuration.cc

```diff
@@ -54,6 +56,31 @@ configuration::configuration()
       "consumer_instance_timeout_ms",
       "How long to wait for an idle consumer before removing it",
       {},
-      std::chrono::minutes{5}) {}
-
+      std::chrono::minutes{5})
+  , client_cache_max_size(
+      *this,
+      "client_cache_max_size",
+      "The maximum number of kafka clients in the LRU cache",
+      {.needs_restart = config::needs_restart::no},
+      10,
+      [](const size_t max_size) {
+          std::optional<ss::sstring> msg{std::nullopt};
+          if (max_size == 0) {
+              msg = ss::sstring{"Client cache max size must not be zero"};
+          }
+          return msg;
+      })
+  , client_keep_alive(
+      *this,
+      "client_keep_alive",
+      "Time in milliseconds that an idle connection may remain open",
+      {.needs_restart = config::needs_restart::no, .example = "30000"},
+      30s,
```

Comment:
I think `5m` is a better default.

Reply:
What's your reasoning? I figured 30s is a good starting place and then, depending on customer feedback, we can change defaults at a later time.

Reply:
Spoke with Ben about this. The reasoning is that a consumer on the proxy has a default poll of ~5min so the connection may be cleaned while the consumer is waiting.